Dev-Corner/slides/tool_integration.html

```diff
@@ -45,65 +45,1274 @@
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### IUC: Intergalactic Utilities Commission
+
+                            - Lobbyist :)
+                            - Host orphan tools: [GitHub](https://github.com/galaxyproject/tools-iuc)
+                            - [IUC-standards](http://galaxy-iuc-standards.readthedocs.io/)
+
+                        </script>
+                    </section>
+                </section>
+
+
+                <!-- PUBLISHING TOOLSHED -->
+                <section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ## Publishing on Toolshed
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Publishing on ToolShed
+
+                            Metadata description: `.shed.yml` file
+
+                            ```
+                            categories:
+                              - Metagenomics
+                              - Graphics
+                              - Phylogenetics
+                            description: "GraPhlAn is a software tool for producing high-quality
+                            circular representations of taxonomic and phylogenetic trees"
+                            homepage_url: "https://bitbucket.org/nsegata/graphlan/overview"
+                            name: graphlan
+                            owner: bebatut
+                            remote_repository_url: "https://github.com/ASaiM/galaxytools/tree/master/tools/graphlan/"
+                            ```
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Publishing on ToolShed
+
+                            Metadata description: `.shed.yml` file
+
+                            ```
+                            graphlan/
+                            ├── graphlan.xml
+                            ├── test_data/
+                                ├── input_tree.txt
+                                ├── png_image.png
+                            ├── .shed.yml
+                            ```
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/planemo-logo.png" alt="Planemo" class="no-border" height=25/>
+                            #### Publishing to the ToolShed
+
+                            <img src="../images/planemo_toolshed_publishing.png" alt="Publishing on ToolShed" class="no-border"/>
+
+                        </script>
+                    </section>
+                </section>
+
```

Comment:
I would have moved the Planemo summary (big picture + checklist) here. What do you think?